### PR TITLE
Store and reload events

### PR DESCRIPTION
**Pull request by tomasdundacek**
Store and reload events whose bindings are loosen during replacement.

### CODE REVIEW

File: app/assets/javascripts/translation_engine/screenshots.js.coffee [hunk 1]
```diff
@@ -11,10 +11,12 @@ class TranslationEngine
   variant: 'desktop'
 
   constructor: (startButton) ->
+    events_ary = @storeBoundEvents()
```
Comment (XeeD) [inline app/assets/javascripts/translation_engine/screenshots.js.coffee]: camelCase in JS land :wink:

Comment (tomasdundacek) [inline app/assets/javascripts/translation_engine/screenshots.js.coffee]: yep :wink:

File: app/assets/javascripts/translation_engine/screenshots.js.coffee [hunk 2]
```diff
@@ -106,6 +108,29 @@ class TranslationEngine
...
+  reBindEvents: (ary) ->
+    for elementEvents in ary
+      $element = $(elementEvents.element)
+      id = $element.attr('id')
+      classes = $element.attr('class')?.split(' ').join('.')
+      tagName = $element.prop('tagName').toLowerCase()
+
+      selector = []
+      selector.push(tagName)
+      selector.push("##{$element.attr('id')}") if $element.attr('id')
```
Comment (tomasdundacek) [inline app/assets/javascripts/translation_engine/screenshots.js.coffee]: use `id` variable from above

**Comment by tomasdundacek** [comment]
don't merge yet pls

**Pull request merged**